Shared/WriteFreely_MultiPlatformApp.swift

```diff
@@ -152,13 +152,13 @@ struct WriteFreely_MultiPlatformApp: App {
     }
 
     private func showLastDraftOrCreateNewLocalPost() {
-        #if os(macOS)
         if model.editor.lastDraftURL != nil {
-            self.model.selectedPost = model.editor.fetchLastDraftFromAppStorage()
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.25) {
```

Comment:
While the app is launching, we need to wait a short period of time for `model.selectedPost` to settle before setting it to the last draft, or else it'll get unset.